pkg/transforms/argoapplication.go

```diff
@@ -73,14 +98,43 @@ func ArgoApplicationResourceBuilder(a *ArgoApplication) *ArgoApplicationResource
 	node.Properties["chart"] = a.Spec.Source.Chart
 	node.Properties["repoURL"] = a.Spec.Source.RepoURL
 	node.Properties["targetRevision"] = a.Spec.Source.TargetRevision
+
 	if a.Spec.Source.TargetRevision == "" {
 		node.Properties["targetRevision"] = "HEAD"
 	}
 
 	// Status properties
-	node.Properties["status"] = a.Status.Health.Status
+	node.Properties["healthStatus"] = a.Status.Health.Status
+	node.Properties["syncStatus"] = a.Status.Sync.Status
+
+	// conditions properties, each condition type is a property
+	for _, condition := range a.Status.Conditions {
+		truncatedMessage := TruncateText(condition.Message, 512)
+		if truncatedMessage > "" {
+			node.Properties["condition"+condition.Type] = truncatedMessage
+		}
+	}
```

Comment:
Status conditions are generic to all resources, so I want to take a closer look at the forma so it's useful in the context of searching for resources matching a given condition.

#### Sample conditions for a Pod resource.
<img width="1087" alt="image" src="https://user-images.githubusercontent.com/4671325/201752400-d5602e11-89de-4e12-866b-9b07237b9f8b.png">

Reply:
I don't want to block this PR either. Maybe we could use a temporary property name like this?
```go
node.Properties["_internalCondition"+condition.Type] = truncatedMessage
```

Reply:
@jlpadilla  Thanks! just updated the PR to the the temporary property name.  We can create a new PR once you figure it out.

Reply:
Thank you! Hopefully we can revisit this soon.